lib/middlewares/mrashify.rb

```diff
@@ -7,8 +7,10 @@
 # which breaks our current implementation of all of the above, hence our
 # custom branch of rash_alt that allows us to use a lesser version of hashie.
 # A PR has been opened to update faraday_middleware...
-# whose current version conflicts with the dependencies in the instagram gem...
-# which was deprecated in 2015 and which only remains in our code because we haven't torn it out yet...
+# whose current version conflicted with the dependencies in a now-removed instagram gem...
+# which was deprecated in 2015.
+# Fully removing this custom middleware will require some additional work
+# and has been put off until a later date.
```

Comment:
Per discussion with @MothOnMars, we may want to remove `mrashify` at some point, but it's not a risk to leave it in and is out of scope for this PR. For now, I've just updated the note at the top of the file for future reference.

Reply:
Sounds good.  I see the https://cm-jira.usa.gov/browse/SRCH-3228 follow on.